BytesOfMan/bch-js
# Removing all API calls, axios, and unit tests related to API calls

With Cashtab running exclusively on eCash's native indexer, chronik, there is no need to use bch-api. However, there is still a need to use some crypto functions included in the bch-js library.

This is a first step toward creating a lighter bch-js that is

1. Focused on XEC
2. Has nothing to do with a server

Test plan
1. Pull down this repo at this branch
2. Install it as a dependency in Cashtab instead of bch-js
3. See if Cashtab works [after modifying for this instead of minimal-slp-wallet, might need to compile this a la minimal-slp-wallet]
